worlds/spelunker/docs/setup_en.md

```diff
@@ -0,0 +1,122 @@
+# Yoshi's Island Archipelago Randomizer Setup Guide
+
+## Required Software
+
+- [Archipelago](https://github.com/ArchipelagoMW/Archipelago/releases).
+
+
+- Hardware or software capable of loading and playing SNES ROM files
+    - An emulator capable of connecting to SNI such as:
+        - snes9x-rr from: [snes9x rr](https://github.com/gocha/snes9x-rr/releases),
+        - BizHawk from: [TASVideos](https://tasvideos.org/BizHawk)
+        - snes9x-nwa from: [snes9x nwa](https://github.com/Skarsnik/snes9x-emunwa/releases)
+        
+        NOTE: RetroArch and FXPakPro are not currently supported.

```

Comment:
```suggestion
        NOTE: RetroArch and FXPakPro are not currently supported. It is not recommended to attempt to play this on console.
```
Mostly since I've seen people try to play the Yoshi's Island one on console a few times and ruin multis, so it makes some sense to more explicitly say not to try console.